scripts/demo.sh

```
@@ -446,3 +446,112 @@ echo -e "  • Check Redis idempotency:   kubectl exec -it redis-0 -- redis-cli"
+        esac
+
+        echo ""
+        echo -e "${CYAN}► Running: $mode_desc${NC}"
+        echo -e "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+
+        go run ./cmd/horizon-demo --timeout "${timeout_ms}ms" --output ./integrity_report.json
+        local exit_code=$?
+
+        if [ -f "./integrity_report.json" ]; then
+            echo ""
+            local verdict=$(jq -r '.verdict' ./integrity_report.json 2>/dev/null)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)